src/tests/validity/mv.ab

```diff
@@ -0,0 +1,5 @@
+unsafe $cd /tmp$
+unsafe $touch a$
+mv "a" "b" failed{}
```

Comment:
The test fails because failed can't be empty

Reply:
I don't think it was the issue as it compiles fine but emits a warning suggesting the use of the  the `unsafe` keyword when the failed block is empty. I've addressed this by explicitly using the `unsafe` keyword in the current implementation.